internal/tiltfile/helm_test.go

```diff
@@ -109,39 +109,40 @@ const exampleHelmV3_0VersionOutput = `v3.0.0`
 const exampleHelmV3_1VersionOutput = `v3.1.0`
 const exampleHelmV3_2VersionOutput = `v3.2.4`
 
+// see https://github.com/tilt-dev/tilt/issues/3788
+const exampleHelmV3_3VersionOutput = `WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/someone/.kube/config
+WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /Users/someone/.kube/config
+v3.3.3+g55e3ca0`
```

Comment:
can you also add a trailing newline to the test case to make sure it still works? since that's how the real output looks